.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: node_js
 
+env:
+  global:
+    - secure: IIEC4WqIlugoTq2N/nrZN9j5v0PU13Mo0XacDOKdr18b3cewDQ1n3eicLjVxU+FZh6U1gb1awdIOJ4bnNCgaeiMs2KjAvI8xxsGFwKn996vT2qmJRlOMyowic4Ru0QmkZ+eiHE1FTSfbG7np0WmRpVX788mgpaGJvj7OrQgzVxM=
```

Comment:
They're encrypted credentials for Travis to talk to Saucelabs:

https://docs.saucelabs.com/ci-integrations/travis-ci/